File: 1-two-sum/1-two-sum.cpp
#include <map>
using namespace std;
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> table;
        for (int i = 0; i < nums.size(); i++) {
            if (table.find(nums[i]) != table.end()) {
                return {table[nums[i]], i};
            } else {
                table[target-nums[i]] = i;
            }
        }
        return {999, 999};
    }
};

File: 1-two-sum/1-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for j in range(len(nums)):
            for i in range(j+1, len(nums)):
                if nums[i] + nums[j] == target:
                    return [j, i]
        